### ISSUE TITLE
/* generated by CMake */   => #include "cocos2d.h"   should it not be "axmol.h"?D

### ISSUE THREAD

**Issue by aismann** [axmolengine/axmol#919]
```
/* generated by CMake */

#pragma system_header
#ifdef __cplusplus
#include "cocos2d.h"
#endif // __cplusplus
```

![image](https://user-images.githubusercontent.com/8652787/195094246-28480602-0e91-4c60-8415-073f679f702e.png)

**Comment by halx99** [comment]
Maybe @crazyhappygame can help you

**Comment by aismann** [comment]
> Maybe @crazyhappygame can help you

And fix it;)

**Comment by rh101** [comment]
Didn't this latest PR fix the issue? https://github.com/axmolengine/axmol/pull/922

**Comment by halx99** [comment]
> Didn't this latest PR fix the issue? #922

Yes
